docs/topics/mta-web-assigning-application-credentials.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-assigning-application-credentials_{context}"]
+= Assigning application credentials
+
+You can assign credentials to one or more applications.
+
+.Procedure
+
+. In *Developer* view, click *Application inventory*.
+. Click the *Analysis* tab.
++
+// ![](/Tackle2/AddingApps/SelectManageCred.png)
+
+. Click the Options menu {kebab} to the right of *Review* and select *Manage credentials*.
```

Comment:
In Analysis Tab there is no Review button we can mention here "**Analyze**" instead 
![Screenshot from 2022-11-03 12-30-43](https://user-images.githubusercontent.com/87791967/199663462-a0276e9e-d848-4201-a7fd-801ec83c4738.png)